### ISSUE TITLE
Add lang attribute to <html> element

### ISSUE THREAD

**Issue by zowiebeha** [lansingcodes/slackin-redirect#11]
We can add `lang="en-US"` as an attribute of the root element. This will improve the page's accessibility for screen reader assisted users. It will fulfill a requirement listed under WCAG Success Criterion 3.1.2, as mentioned in the following document:

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/lang#accessibility_concerns

Notable reason for why we define it on the root specifically:
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/lang#inheritance
